README.md

```diff
@@ -1,117 +1,69 @@
-# SkyPilot
+
+![logo](docs/source/images/SkyPilot-logo-wide.png)
```

Comment:
We can use some markdown magic to limit the width?  Currently it feels too wide compared to size of the text.  E.g., https://github.com/ray-project/ray/

Reply:
Done! Set to `80%`.